src/clj/villagebook/organisation/handlers.clj

```diff
@@ -2,24 +2,26 @@
   (:require [ring.util.response :as res]
             [clojure.edn :as edn]
             [villagebook.organisation.db :as db]
```

Comment:
Should probably offload db access to the model namespace.

Typically the namespaces should form layers.
handler -> http request / response parsing and cleanup
model -> business logic
db -> db